modular_skyrat/modules/sec_haul/code/misc/ai_module.dm

```diff
@@ -1,12 +0,0 @@
-/obj/item/ai_module/core/full/armadyne_safeguard
```

Comment:
Honestly, probably don't need to delete the law module, just changing the default config should be enough.

Having the laws on standby would be good for events, maybe